src/NHibernate.Test/NHSpecificTest/NH2030/Fixture.cs

```diff
@@ -15,7 +15,7 @@ public class Fixture
 		[Test]
 		public void GetTypeWithLenShouldBeThreadSafe()
 		{
-			object sync = new object();
+			Lock sync = new Lock();
```

Comment:
What is the point of changing the locking mechanism of existing tests?

Reply:
Consistency maybe; it also helps in case something breaks with regards to the lock object. Can revert if you want.

Reply:
Reverted